updatecli/updatecli.d/letsencrypt_cert.ci.yaml

```diff
@@ -0,0 +1,61 @@
+name: "Bump lets_encrypt_dns_challenged_domains `expiry` date for cert.ci"
```

Comment:
Could you update the title to reflect the intent? 

- `lets_encrypt_dns_challenged_domains` is not clear about which date is renewed here. It cannot be the certificate date otherwise it would make no sense as letsencrypt takes care of autorenewal.

(of course it also implies to update the reste of the PR ;) )